src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,33 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-gap: 48px;
+  flex-wrap: wrap;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, auto);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, auto);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, auto);
+    }
+  }
+
+  &__card {
+    display: grid;
```

Comment:
Do you really need display: greed for card?

Reply:
I've just tried to get more practice:)